mayastor/src/bdev/nexus/nexus_bdev.rs

```diff
@@ -1080,12 +1100,40 @@ pub async fn nexus_create(
         return Ok(());
     }
 
+    let (min_cntlid, max_cntlid, resv_key) =
+        nvme_params.unwrap_or((NVME_MIN_CNTLID, NVME_MAX_CNTLID, 0x1234_5678));
+    if min_cntlid < NVME_MIN_CNTLID
+        || min_cntlid > NVME_MAX_CNTLID
+        || max_cntlid < NVME_MIN_CNTLID
+        || max_cntlid > NVME_MAX_CNTLID
+        || min_cntlid > max_cntlid
```

Comment:
We perhaps should check these numbers before pass the RPC request to mayastor?

Reply:
I prefer to validate at the lowest level to avoid any codepath in the future that manages to bypass that checking.